pkg/chunk/cassandra/storage_client_test.go

```diff
@@ -75,6 +75,37 @@ func TestConfig_setClusterConfig_authWithPasswordAndPasswordFile(t *testing.T) {
 	assert.Error(t, cfg.Validate())
 }
 
+func TestConfig_setClusterConfig_clientSSL(t *testing.T) {
+	cfg := defaultConfig()
+	cfg.SSL = true
+	cfg.CAPath = "testdata/example.com.ca.pem"
+	cfg.CertPath = "testdata/example.com.pem"
+	cfg.KeyPath = "testdata/example.com-key.pem"
+	require.NoError(t, cfg.Validate())
+
+	cqlCfg := gocql.NewCluster()
+	err := cfg.setClusterConfig(cqlCfg)
+	require.NoError(t, err)
+	assert.NotNil(t, cqlCfg.SslOpts)
+	assert.Len(t, cqlCfg.SslOpts.Certificates, 1)
+}
+
+func TestConfig_setClusterConfig_clientSSLWithOnlyCertificatePath(t *testing.T) {
+	cfg := defaultConfig()
+	cfg.SSL = true
+	cfg.CAPath = "testdata/example.com.ca.pem"
+	cfg.CertPath = "testdata/example.com.pem"
+	assert.Error(t, cfg.Validate())
```

Comment:
Could you assert on the exact error message, please?

_Same comment applies to the next test._